marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/ModelControllerTest.java

```diff
@@ -60,26 +69,114 @@ private void updateModelInfo() {
     }
 
     private void updateModelEntityTypes() {
+        // Loading unrelated indexes so that we can check for them after updating entity model
```

Comment:
I believe you need to call Installer.applyDatabasePropertiesForTests after each test here because you're modifying the config of the staging/final databases, which could break other tests.

Reply:
Updated in here b6777af